exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Game.java

```diff
@@ -16,8 +17,9 @@
     
     int currentPlayer = 0;
     boolean isGettingOutOfPenaltyBox;
-    
-    public  Game(){
+	private int maxNumberOfCoins;

```

Comment:
This looks like it wants to be a constant?  If so should probably be final static and be ALL_IN_UPPER_CASE.